p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1371,8 +1371,9 @@ do not require terminating semicolons).
 Various constructs act as namespaces that create local scopes for
 names:
 
-- Derived type declarations (```struct```, ```header```, ```enum```)
-  introduce local scopes for the field names
+- Derived type declarations (```struct```, ```header```,
+  ```header_union```, ```enum```) introduce local scopes for the field
```

Comment:
you want to keep this on a single line, even if it is longer than 80 chars. Otherwise madoko interprets header_union as a language and emits a warning. Sometimes also messes up formatting.